docs/UserGuide.md

```diff
@@ -3,13 +3,14 @@ layout: page
 title: User Guide
 ---
 
-_bookmark_ is a desktop application to manage your indulgent bookworm habits! Use _bookmark_ to track your book reading progress 
-or pen down thoughts of mindboggling fan theories about your favourite characters!
+_bookmark_ is a desktop application for readers to manage their reading progress.
 
-_bookmark_ allows you to add and update your beloved books, track your reading progress using our bookmark and even add reading goals and notes to your chosen books!
+_bookmark_ allows you to easily track your **reading progress** so you'll never need to use a physical bookmark or
+lose track of where you stopped reading again. With many interesting features like **note taking** and **goal setting**,
+_bookmark_ will be your faithful reading companion, helping to make reading even more interactive and exciting.
 
-Guess what? _bookmark_ also uses the special Command Line Interface style so if you love to type, you're in luck! So 
-get cracking and start saving your best reads and ideas into _bookmark_, Type away!~
+This user guide introduces what you can do with _bookmark_ and how it can benefit you. With this guide,
+you will be ready to use _bookmark_ to enhance your reading experience!
```

Comment:
Maybe we can do "This user guide introduces you to the many features of _bookmark_ and how you can benefit from them. With this guide, you will be ready to use _bookmark_ to enhance your reading experience.  [remove the exclamation mark]